zebra-chain/src/transaction.rs

```diff
@@ -457,6 +488,14 @@ impl Transaction {
                 block::Height(0) => None,
                 block::Height(expiry_height) => Some(block::Height(*expiry_height)),
             },
+            #[cfg(feature = "tx_v6")]
+            Transaction::V6 { expiry_height, .. } => match expiry_height {
+                // Consensus rule:
```

Comment:
```suggestion
                // # Consensus
                //
```

We use a special comment for consensus rules so we can find them all searching for the `# Consensus` string in the codebase.

For example: https://github.com/ZcashFoundation/zebra/blob/main/zebra-chain/src/transparent.rs#L210